crawl-ref/source/game-options.cc

```diff
@@ -7,9 +7,38 @@
 
 #include "game-options.h"
 #include "options.h"
+#include "menu.h"
+#include "message.h"
 #include "misc.h"
+#include "prompt.h"
+#include "scroller.h"
 #include "tiles-build-specific.h"
 
+// Return a list of the valid field values for _curses_attribute(), excluding
+// hi: and hilite: values, which are all duplicates.
+// This is called after initialisation to ensure that colour_to_str() works.
+static map<unsigned, string>& _curses_attribute_map()
+{
+    static map<unsigned, string> list;
```

Comment:
`list` is a pretty unfortunate name here - it's not a list, it's a map, and the name `list` gives the reader no information about what's in it. Perhaps call this `attrs_to_strings` or similar.

Reply:
I've renamed the variable (to "ret" - the function's just a constructor, so that seemed like the best name for the thing it returns), and rearranged the initialiser list as you said above.